packml_msgs/msg/Stats.msg

```diff
@@ -0,0 +1,29 @@
+# This message contains machine agnostic information that is extracted from PackML
+# standard states
+
+std_msgs/Header stamp       # Duration of statistics calculation
+std_msgs/Duration duration      # Duration over which statistics are calculated
```

Comment:
I just wanted to point out the discrepancy. Conversions to PackML's `Time` will result in loss of precision then, but as far as that is clear, fine with me.